### ISSUE TITLE
GetSelectedRows usage

### ISSUE THREAD

**Issue by alvarolm** [mattn/go-gtk#275]
I wanto iterate over the selected rows, so far I haven't been able to do it.
```
fmt.Println("rows selected", sel.CountSelectedRows()) 

plist := sel.GetSelectedRows(&model)

list := glib.ListFromNative(unsafe.Pointer(plist))

list.ForEach(func(d unsafe.Pointer, v interface{}) {
	fmt.Println("cast to ...?", d, v)
})		
```
outputs:
```
rows selected 3
cast to ...? 0x1edc430 <nil>
cast to ...? 0x1ef6b40 <nil>
cast to ...? 0x1ef6e10 <nil>
```

Isn't needed something like ?
```
func TreePathFromNative(val interface{}) *GtkTreePath {
	return &GtkTreePath{*to_GtkTreePath(val)}
}
```
(https://github.com/mattn/go-gtk/issues/114)

**Comment by andlabs** [comment]
[Yes, the list element type is GtkTreePath.](https://developer.gnome.org/gtk2/stable/GtkTreeSelection.html#gtk-tree-selection-get-selected-rows)

**Comment by alvarolm** [comment]
with https://github.com/mattn/go-gtk/pull/276 works perfectly, this pr only introduces 
`TreePathFromNative(value unsafe.Pointer) *TreePath` so it will not break the current implementation.

**Comment by mattn** [comment]
Closed by #276
